packs/feat-effects/effect-seedpod.json

```
@@ -4,7 +4,7 @@
     "name": "Effect: Seedpod",
     "system": {
         "description": {
-            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Seedpod]</p>\n<p>On a critical hit, a seedpod bursts, issuing forth a tangle of vegetation that imposes a -10-foot circumstance penalty on the target's Speed for 1 round.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Seedpod]</p>\n<p>You have a -10-foot circumstance penalty on the your Speed.</p>"
```

```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Seedpod]</p>\n<p>You take a -10-foot circumstance penalty to your Speed.</p>"
```
Update the RE to use `land-speed`